src/modules/InvolvementAPI.js

```diff
@@ -13,11 +13,27 @@ class InvolvementAPI {
       .catch((error) => error));
   }
 
+  async postLike(gameId) {
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ item_id: gameId }),
+    };
+
+    try {
+      const response = await fetch(`${process.env.INVOLVEMENT_API_BASE_URL}${process.env.APP_ID}/likes`, requestOptions)
+        .catch((error) => error);
+      this.responseStatus = response.status;
+    } catch (error) {
+      this.responseStatus = error;
+    }
+    return this.responseStatus;
+  }
+
```

Comment:
You could leave out they .catch line t line 25 as any error in a try.....catch block is sent to the catch block{} regardless of where the error has come from in the try{} block. All exceptions are thrown to the catch block 

- Nice tutorial for reference: `https://www.javascripttutorial.net/javascript-try-catch/`